tools/ds-identify

```diff
@@ -1487,6 +1487,13 @@ dscheck_VMware() {
     return "${DS_NOT_FOUND}"
 }
 
+dscheck_Outscale() {
+    local product_name="${DI_DMI_PRODUCT_NAME}"
+    local sys_vendor="${DI_DMI_SYS_VENDOR}"
+    test "${product_name}" = "3DS Outscale VM"  &&  test "${sys_vendor}" = "3DS Outscale" && return ${DS_FOUND}
```

Comment:
Just for consistency's sake, can we use `if` instead of test here? Also quote `${NOT_FOUND}` as well as `${DS_NOT_FOUND}` on the next line.

Reply:
You're right.